lib/chef/audit/runner.rb

```diff
@@ -96,6 +97,7 @@ def configure_rspec
         RSpec.configure do |c|
           c.color = Chef::Config[:color]
           c.expose_dsl_globally = false
+          c.backtrace_exclusion_patterns << Regexp.new(Chef::Config.platform_specific_path('/opt/chef'))
```

Comment:
I think you could introspect `__FILE__` and use the first couple path segments from that to setup the exclusion pattern